ui/pages/swaps/select-quote-popover/quote-details/quote-details.js

```diff
@@ -80,7 +80,7 @@ const QuoteDetails = ({
       <div className="quote-details__row quote-details__row--high">
         <div className="quote-details__detail-header">
           <img
-            src="./images/logo/metamask-fox.svg"
+            src="/images/logo/metamask-fox.svg"
```

Comment:
this may break storybook. why was it changed?

Reply:
Don't recall why I made this change -- I've reverted this change.